gutenberg.php

```diff
@@ -161,3 +161,16 @@ function gutenberg_rest_nonce() {
 	exit( wp_create_nonce( 'wp_rest' ) );
 }
 add_action( 'wp_ajax_gutenberg_rest_nonce', 'gutenberg_rest_nonce' );
+
+
+/**
+ * Exposes the site icon url to the Gutenberg editor through the WordPress REST API.
+ */
+function register_site_icon_url_setting() {
+	register_setting('general', 'site_icon_url', array(
```

Comment:
we should update this to match the WordPress php style standards. I think you can also find issues locally by running the lint php script. (`npm run lint-php`) Basically you just need to add spaces before and after the function call

Reply:
PHPCBF is our friend:

```
composer install
vendor/bin/phpcbf gutenberg.php
```

Reply:
(There are integrations for popular editors. I like to flip on format-on-save :grimacing: )